_15thnight/api/account.py

```diff
@@ -74,16 +74,17 @@ def forgot_password():
                 user.reset_created_at < datetime.utcnow() - reset_token_life):
             user.generate_reset_token()
         user.save()
+
         link = '%sreset-password/%s/%s' % (
-            url_for('index', _external=True), user.email, user.reset_token
-        )
+            url_for('index', _external=True), user.email, user.reset_token)
```

Comment:
Many times you have mentioned (specifically with imports anyway) that if all the code, including final paren, can fit on one line, it should. That is why I changed this.